projects/hmpps-auth-and-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/controller/UserController.kt

```diff
@@ -23,24 +23,24 @@ import uk.gov.justice.digital.hmpps.service.UserService
 @Tag(name = "User details")
 class UserController(private val userService: UserService) {
     @GetMapping(value = ["/user/{username}"])
-    @PreAuthorize("hasAnyRole('ROLE_DELIUS_USER_AUTH', 'ROLE_DELIUS_USER_DETAILS')")
+    @PreAuthorize("hasAnyRole('ROLE_DELIUS_USER_AUTH','ROLE_DELIUS_USER_DETAILS','PROBATION_API__HMPPS_AUTH__AUTHENTICATE')")
     @Operation(description = "Get user details. Requires `ROLE_DELIUS_USER_AUTH` or `ROLE_DELIUS_USER_DETAILS`.")
     fun getUserDetails(@PathVariable username: String) = userService.getUserDetails(username)
         ?: throw NotFoundException("User", "username", username)
 
     @GetMapping(value = ["/user/details/{userId}"])
-    @PreAuthorize("hasAnyRole('ROLE_DELIUS_USER_AUTH', 'ROLE_DELIUS_USER_DETAILS')")
+    @PreAuthorize("hasAnyRole('ROLE_DELIUS_USER_AUTH','ROLE_DELIUS_USER_DETAILS','PROBATION_API__HMPPS_AUTH__AUTHENTICATE')")
     @Operation(description = "Get user details by Id. Requires `ROLE_DELIUS_USER_AUTH` or `ROLE_DELIUS_USER_DETAILS`.")
     fun getUserDetailsById(@PathVariable(required = true) userId: Long) = userService.getUserDetailsById(userId)
         ?: throw NotFoundException("User", "userId", userId)
 
     @GetMapping(value = ["/user"])
-    @PreAuthorize("hasAnyRole('ROLE_DELIUS_USER_AUTH')")
+    @PreAuthorize("hasAnyRole('ROLE_DELIUS_USER_AUTH','PROBATION_API__HMPPS_AUTH__AUTHENTICATE')")
     @Operation(description = "Get users by email. Requires `ROLE_DELIUS_USER_AUTH`.")
     fun getUsersByEmail(@RequestParam email: String) = userService.getUsersByEmail(email)
 
     @PostMapping("/user/{username}/password")
-    @PreAuthorize("hasRole('ROLE_DELIUS_USER_AUTH')")
+    @PreAuthorize("hasAnyRole('ROLE_DELIUS_USER_AUTH','PROBATION_API__HMPPS_AUTH__AUTHENTICATE')")
```

Comment:
Maybe a different role for this one too?
```suggestion
    @PreAuthorize("hasAnyRole('ROLE_DELIUS_USER_AUTH','PROBATION_API__HMPPS_AUTH__UPDATE_PASSWORD')")
```